### DISCUSSION TITLE
Comment

### DISCUSSION THREAD

**Discussion by lancedouglas1** [toeverything/AFFiNE#6148] [Ideas]
Hi AFFiNE team,

I've been exploring the functionalities of Affine Pro and I find the platform's potential exciting. There's one feature I've found lacking that I believe could greatly enhance the platform's utility (_`unfortunately, it is a critical feature for my company and clients to adopt AFFiNE`_):

**Commenting Feature:** In this age of collaborative work, enabling users to comment on specific sections of their content has become critical. The ability to highlight specific text and embed a comment makes brainstorming, problem-solving, and collaborative thinking a breeze - readers can effortlessly pinpoint where they need to focus their attention. Adding commenting functionality, much like what's found in other collaborative tools, would significantly enrich AFFiNE's offering.

I understand that adding features takes time and resources. However, I'm confident this addition would simultaneously bolster user experience and make AFFiNE more appealing to a broader range of collaborative groups. 

Thanks for considering this suggestion.  I've begun sponsoring AFFiNE because I do believe in its potential.

**Comment by lancedouglas1** [comment]
I found this related post on blocksuite: https://github.com/toeverything/blocksuite/issues/3121

**Comment by MurzNN** [comment]
Seems comments are already implemented now, right?
